internal/newsfeed/news_feed_manager.go

```diff
@@ -0,0 +1,126 @@
+package newsfeed
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/mmcdole/gofeed"
+	gocommon "github.com/status-im/status-go/common"
```

Comment:
you need to add a blank line between each different github account from which stuff is imported